packages/cli/src/lingui-extract-template.ts

```diff
@@ -24,11 +22,6 @@ export default async function command(
     process.env.BABEL_ENV = "development"
   }
 
-  // We need macros to keep imports, so extract-messages plugin know what componets
-  // to collect. Users usually use both BABEN_ENV and NODE_ENV, so it's probably
-  // safer to introduce a new env variable. LINGUI_EXTRACT=1 during `lingui extract`
-  process.env.LINGUI_EXTRACT = "1"
-
```

Comment:
That flag was used to tell babel-macro plugin to not delete macro imports. 
That actually don't need. Because extractor analyze code AFTER macro transformation and lokiing for `i18n._` and `/*i18n*/` statements, not a macro calls.